pages/common/zpool.md

```diff
@@ -4,30 +4,28 @@
 
 - Show the configuration and status of all ZFS zpools:
 
-`zpool status [{{poolname}}]`
+`zpool status`
 
 - Check a ZFS pool for errors (verifies the checksum of EVERY block). Very CPU and disk intensive:
 
-`zpool scrub {{poolname}}`
+`zpool scrub {{pool_name}}`
 
 - List zpools available for import:
 
 `zpool import`
 
-- Import a zpool, optionally specifying a new name:
+- Import a zpool (optionally renaming it):
 
-`zpool import {{poolname}}`
-`zpool import {{poolname}} {{newpoolname}}`
+`zpool import {{pool_name}} {{[new_pool_name]}}`
 
 - Export a zpool (unmount all filesystems):
 
-`zpool export {{poolname}}`
+`zpool export {{pool_name}}`
 
 - Show the history of all pool operations:
 
-`zpool histrory {{poolname}}`
+`zpool histrory {{pool_name}}`
 
 - Create a mirrored pool:
 
-`zpool create {{poolname}} mirror {{disk1}} {{disk2}}`
-`zpool create {{poolname}} mirror {{disk1}} {{disk2}} mirror {{disk3}} {{disk4}}`
+`zpool create {{pool_name}} mirror {{disk1}} {{disk2}} mirror {{disk3}} {{disk4}}`
```

Comment:
Not sure leaving only the longer example is a good choice, but I'll leave it to those who actually know this command to decide.